data/soapvalidator/MailClient/Calendar/Bugs/bug45757.xml

```diff
@@ -175,7 +175,7 @@
 
 	<t:property name="server.zimbraAccount" value="${user3.server}"/>
 
-    
+    <t:delay sec="5"/>
```

Comment:
Please add this comment on all these lines so it is easy to tidy up in the future.  <!-- Wait for Solr commit to happen -->